src/components/PieChart/PieChart.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+import React, { HTMLAttributes } from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend,
+  Filler,
+  ChartData,
+  ChartOptions,
+} from 'chart.js';
+import { Doughnut } from 'react-chartjs-2';
+import { cn } from '@/lib/utils';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend,
+  Filler
+);
```

Comment:
P2; 현재 chart.js에서 가져오는 코드 크기가 크더라구요. 혹시 이 부분에서 사용하지 않는 코드를 줄이면 문제가 생길까요? 
제가 알아본 대로 일부 미사용 코드를 제거해봤고 테스트 시 제 환경에서는 문제 없었는데 확인 한 번 부탁드립니다!
이렇게 줄일 시 여전히 크긴 하지만 129.9k 로 줄어듭니다! suggestion 남깁니다!
```suggestion
import {
  Chart as ChartJS,
  ArcElement,
  Tooltip,
  Legend,
  ChartOptions,
  ChartData,
} from 'chart.js';
import { Doughnut } from 'react-chartjs-2';
import { cn } from '@/lib/utils';

ChartJS.register(ArcElement, Tooltip, Legend);
```
![image](https://github.com/user-attachments/assets/37e9321a-a5b4-45c5-803b-b1575ae18772)

Reply:
확인해보겠습니다!

Reply:
삭제 후 154.2k (gzipped: 53.1k)에서 129.9k (gzipped: 44.4k)로 줄었습니다!

![스크린샷](https://github.com/user-attachments/assets/b4a0a52d-5476-4c4b-a5c9-a377cbb683e0)

Reply:
우와!! 혹시 이거 용량은 어떻게 볼 수 있나요?

Reply:
@kim-hyunjoo Import Cost라는 vsc 라이브러리입니다!
https://marketplace.visualstudio.com/items?itemName=wix.vscode-import-cost